build.sbt

```diff
@@ -99,6 +95,7 @@ lazy val modules: List[CompositeProject] = List(
   doobie,
   skunk,
   generic,
+  docs,
```

Comment:
Typically I don't add the documentation site project to the root, but it's probably fine.